app/models/user.rb

```diff
@@ -8,4 +9,11 @@ class User < ApplicationRecord
   has_many :answers
   has_many :weekly_activities
   enum :base_office, {braga: 0, coimbra: 1}
+  after_commit :add_default_picture, on: [:create]
+
+  private def add_default_picture
+    unless profile_pic.attached?
```

Comment:
```suggestion
    return if profile_pic.attached?
```

This way you don't have to think about conditionals inside of the method. If it's not a valid "state" we immediately return